google/cloud/bigtable/benchmarks/benchmark_test.cc

```diff
@@ -177,7 +177,8 @@ TEST(BenchmarkTest, PrintCsv) {
   // fairly minimal.
 
   // The output includes the version and compiler info.
-  EXPECT_NE(std::string::npos, output.find(bigtable::version_string()));
+  EXPECT_NE(std::string::npos,
```

Comment:
Consider adding a TODO to replace this with:

```cpp
EXPECT_THAT(output, HasSubstr(google::cloud::bigtable::version_string()));
```

and similarly below.

Reply:
I created #653, but dropping TODOs require me to search for all the places where we do this, which is half the effort to fix that bug.